shared/desktop/renderer/index.js

```diff
@@ -93,6 +93,13 @@ function setupApp(store) {
     })
   })
 
+  // After a delay, see if we're connected, and try starting keybase if not
+  setTimeout(() => {
```

Comment:
if that's the case then i'd argue we just don't make a listener and helper on engine and instead just check to see if we've actually ever connected and call this send immediately. so i'd kill `listenOnNotConnected` and add some helper to engine to expose `this._hasConnected && this._rpcClient.transport.needsConnect`, like `engine.hasEverConnected` or something